patterns/1-initial/creating_an_innersource_strategy.md

```diff
@@ -0,0 +1,78 @@
+## Title
+
+Creating an InnerSource Strategy
+
+## Patlet / Summary
+
+Sometimes, it is difficult to convince people of the relevance of InnerSource for your organization and/or to get support from management. Creating an InnerSource strategy, that connects your InnerSource approach and activities to the goals and the overall strategy of your organization, can help in this regard.
+
+## Problem
+
+Despite an organization has an InnerSource program, it is challenging to establish InnerSource in that organization. Missing management support and low awareness in some or many development teams are among the reasons. Potentially, the InnerSource program itself is missing clear goals and/or approaches how to achieve those.
+
+## Context
```

Comment:
I turned that paragraph into a list of bullet points, but did not change the content since I am not sure what is required here. At least on a high level, the paragraph explains where the problem exists, IMO. It could be described in further detail. Is that what you mean or would you expect different content?

Reply:
@misappi we misunderstood each other here. I meant the "Context" section, not the "Problem" section.

Reply:
OK, got it. I extended the context paragraph and converted it into a list of bullet points.

Reply:
@misappi left some comments related to the Context/Forces inline.

Do you want to revert the Problem section to its previous state? Typically we just use prose and no bullets there.